### ISSUE TITLE
QOL: Grab limit setting.

### ISSUE THREAD

**Issue by watthefak** [naelstrof/KoboldKare#67]
It's extremely easy to mistake the "grab +1 upgrade" for a bug. I restarted my game thinking something broke when I kept grabbing two Kobolds any time I tried to pose them.

As a solution and indicator, I would suggest another button with a hand and a number on the bottom-left with a keybind (maybe V or B?) that cycles 1-2-3 between how many items you pick up when grabbing.

The meat mechanic can stay the same, but once you unlock grabbing 2, let me easily switch back to grabbing just 1 when I want to pose stuff.

**Comment by naelstrof** [comment]
Yeah this is important, it's not clear that strength is a kobold stat, or that it can be increased, or that it's a genetic trait.

It's also awfully inconvenient, but only sometimes. I'll try to address it soon!
